views/includes/table_not_fillable_fr.pug

```diff
@@ -3,7 +3,7 @@ if item_list.length > 0
     thead
       tr.active
         th.col-sm-6(scope='col') Clause de la norme EN 301 549
-        th.col-sm-4(scope='col') Détermination de la conformité
+        th.col-sm-4(scope='col') Relation avec les déclarations de performance fonctionnelle
     tbody
       each item in item_list
         tr
```

Comment:
Perhaps I just missed it, but did we put in the French equivalent of "No user impact mapping" in this file?